class-vip-support-user.php

```diff
@@ -629,6 +629,38 @@ protected function get_trigger_resend_verification_url() {
 		return add_query_arg( array( self::GET_TRIGGER_RESEND_VERIFICATION => '1' ) );
 	}
 
+	/**
+	 * Get list of emails for current Mentees from Zendesk
+	 *
+	 * @return array Array of emails for current Mentees
+	 */
+	protected function get_mentees() {
+		$cached_emails = wpcom_vip_cache_get( 'vip_mentees' );
+
+		if ( $cached_emails ) {
+		 return $cached_emails;
+		}
+
+		$url = VIP_ZENDESK_MENTEE_GROUP_URL;
+		$response = wp_remote_get( $url, array(
+			'method' => 'GET',
+			'blocking' => true,
+			'headers' => array(
+				'Content-Type' => 'application/json; charset=utf-8',
+				'Authorization' => 'Basic ' . base64_encode( VIP_ZENDESK_EMAIL . '/token:' . VIP_ZENDESK_TOKEN ) ),
+			)
+		);
+
+		$decoded_response = json_decode( $response[ 'body' ] );
+		$allowed_emails = [];
+
+		foreach ( $decoded_response->users as $user ) {
```

Comment:
`json_decode()` can fail, in which case it will return `null`, so we'll need to handle that condition.